charts/hedera-mirror-common/templates/configmap-test-rest-env.yaml

```
@@ -0,0 +1,9 @@
+{{- if .Values.testkube.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-rest-env
```

Needs to include the release name. Can drop env since there's only one. Same comment for web3 and anywhere that references these configmaps.

```suggestion
  name: {{ include "hedera-mirror-common.fullname" . }}-test-rest
```